### ISSUE TITLE
Publish doesn't show up for changes made while on the page for a single file

### ISSUE THREAD

**Issue by praveen-srinivasan** [Reviewable/Reviewable#362]
Hi,
If I send someone a link to a single file in Reviewable and they approve the file, the UI doesn't seem to show a "PUBLISH" button in the upper right corner, but if they open the page for the pull request, the publish button shows up. Would it be possible to make the publish button show on the single file page?

**Comment by pkaminski** [comment]
Yes, for historical reasons (and to lighten the load on the single-file page as much as possible) there are two separate UIs for these modes.  This will go away eventually -- but not soon -- as I make some major changes to the app to improve overall performance.  In the meantime, I don't recommend sending single-file links around.  (Though note that there's a small up-arrow in the toolbar to make it convenient to get back to the main review page.)

**Comment by pkaminski** [comment]
We'll be able to fix this once #261 is implemented.
